assets/js/modules/analytics-4/components/common/EnhancedMeasurementSwitch.js

```diff
@@ -45,22 +45,20 @@ export default function EnhancedMeasurementSwitch( {
 	onClick,
 	disabled = false,
 	loading = false,
+	formName = ENHANCED_MEASUREMENT_FORM,
```

Comment:
Making this flexible makes it simpler to integrate into contexts with existing forms like AccountCreate and is more consistent with existing uses where we have separate form state for setup+settings and account creation.